web/src/components/DisputeView/DisputeCardView.tsx

```diff
@@ -56,7 +61,7 @@ interface IDisputeCardView {
 const DisputeCardView: React.FC<IDisputeCardView> = ({ isLoading, ...props }) => {
   return (
     <Link to={`/cases/${props?.disputeID?.toString()}`}>
-      <StyledCard hover>
+      <StyledCard>
```

Comment:
I like more the old dropshadow effect, rather than changing the background color 🤔 If we want to make it more evident, maybe add this in line 22 to 26. wdyt?
```
transition: scale 0.1s;
&:hover {
  scale: 1.05;
}
```
![image](https://github.com/user-attachments/assets/949dd5d9-8a08-4639-b8e6-d4183fb1ec52)

Reply:
Imo highlighting its better, gives more dopamine. Polymarket and X do it too

the dropshadow effect its almost not visible in dark mode, (and I don't even see it in light mode), we could add it in addition of this one?

im not sure about scaling either, I will experiment a bit.

Reply:
yeah, no strong opinion on this, we can see how it looks if we maintain both effects, but if doing the background change is the way to go, it's fine 🙌🏻